### ISSUE TITLE
General question regarding train/test error. 

### ISSUE THREAD

**Issue by RebWeb** [DeepLabCut/DeepLabCut#812]
Hi everyone, 

this is just a very general question about refining an re-evaluating the trained network. I just did a refinement (I relabeled 3 videos) and trained my network again (40'000 iterations). After evaluation the refined network, I recognized that my train and test errors are almost 4x as high as after the first training (approx. 2'000000 iterations). I am not sure if I did something wrong when refining and merging my datasets or if I just don't understand the technical basics of the refinement and the corresponding evaluation..? 

Thanks for your help!

**Comment by jeylau** [comment]
Hello @RebWeb, did you point to the latest snapshot in 'init_weights' in the pose_cfg.yaml before re-training?

**Comment by RebWeb** [comment]
Hi @jeylau
No, I only changed the "multi-step" segment in the new pose_cfg.yaml, I did not adjust the init_weights path (c:\users\deeplabcut\anaconda3\envs\dlc208_1\lib\site-packages\deeplabcut\pose_estimation_tensorflow\models\pretrained\resnet_v1_50.ckpt).. I thought this will be updated automatically?

**Comment by jeylau** [comment]
The network is re-trained from scratch by default (e.g., in the case settings in the config.yaml have changed, or to avoid propagating labeling errors present in the original dataset). I would be very careful changing the learning rate schedule though.

**Comment by RebWeb** [comment]
Ok, thanks for your fast reply. So you wouldn't recommend to go with fewer iterations to refine/retrain the network?

**Comment by jeylau** [comment]
Aw sure, and you can also pass it directly to `train_network(maxiters=)`. I thought you had altered the learning rates, which does require more caution 😃

**Comment by AlexEMG** [comment]
That is a bit odd though -- have you added many images? (otherwise you would assume the train/test error is dominated by the original set in which case the errors should be comparable)

**Comment by RebWeb** [comment]
Hi @AlexEMG I trained the network on 75 images and added another 30 images during the refinement step...
